libpod/options.go

```diff
@@ -1339,17 +1340,46 @@ func WithCommand(command []string) CtrCreateOption {
 // This creates a container from a directory on disk and not an image.
 func WithRootFS(rootfs string) CtrCreateOption {
```

Comment:
Can we move the parsing higher in the stack? This would be a lot cleaner if it was `WithRootFS(rootfs string, overlay bool) ...`

Reply:
@mheon this is resolved.